distribution/docs/src/main/resources/_contents/_installing/firewall-config-contents.adoc

```diff
@@ -0,0 +1,34 @@
+=== Firewall configuration
+Below is a table listing all of the ports that ${branding} uses and a description of what they are used for. Firewalls will need to be configured to open these ports in order for external systems to communicate with ${branding}.
+.Port list
```

Comment:
This caption needs a blank line preceding it, it gets swallowed by the previous paragraph